include/termcolor/termcolor.hpp

```diff
@@ -9,9 +9,17 @@
 //! :license: BSD, see LICENSE for details
 //!
 
+#pragma once
+
 #ifndef TERMCOLOR_HPP_
 #define TERMCOLOR_HPP_
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable: 4800) //about force cast int to bool
```

Comment:
Hm.. Is it fixable? According to MSDN [C4800] is about assigning `int` to `bool`. The only place I found that might raise that warning is `is_atty`'s return statements. It seems that both Windows' [_isatty] and Linux's [isatty] return `int` while termcolor's `is_atty` returns `bool`.

If that's a case I think it'd be better to use `static_cast<bool>(...)` on return value.


[C4800]: https://msdn.microsoft.com/en-us/library/b6801kcy.aspx
[_isatty]: https://msdn.microsoft.com/en-us/library/f4s0ddew.aspx
[isatty]: http://linux.die.net/man/3/isatty